framework/keyring-interface.md

```diff
@@ -79,7 +83,7 @@ The key provider ID MUST be a binary value
 and SHOULD be equal to a UTF-8 encoding of the key namespace.
 
 This value MUST NOT be "aws-kms"
```

Comment:
Issue/Suggestion: The new MPL introduces the
["aws-kms-hierarchy"](https://github.com/aws/aws-cryptographic-material-providers-library-java/blob/main/AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/Keyrings/AwsKms/Constants.dfy#L18-L23) & ["aws-kms-rsa"](https://github.com/aws/aws-cryptographic-material-providers-library-java/blob/main/AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/Keyrings/AwsKms/Constants.dfy#L25C21-L29) key provider IDs.

```suggestion
This value MUST NOT be or start with "aws-kms"
```